combat-trainer.lic

```diff
@@ -3727,6 +3738,9 @@ class GameState
     @target_increment = settings.combat_trainer_target_increment
     echo("  @target_increment: #{@target_increment}") if $debug_mode_ct
 
+    @strict_weapon_stance = settings.strict_weapon_stance
```

Comment:
It's minor, but I don't think this needs to be in `game_state` if it's only going to be used once by a single process. I'd rather have it pulled out in `SetupProcess::initialize` (see line 10-ish).

Reply:
Ah, gotcha. I was following the pattern of `current_weapon_stance` to define the new variable. I can switch it to be a class variable in `SetupProcess` instead.